src/components/Content/ContentChemistry.vue

```diff
@@ -55,7 +55,12 @@
               class="mx--2"
               :src="require('@/assets/art/chemistry/chemOil.png')"
             />
-            <b>Bases</b> that make it up.
+            <b>Bases</b> that make it up, and sometimes you will a special vessel made with

```

Comment:
I don't really like the wording here. 
The "special vessel" is really only relevant for the first chem, but it sounds like the sentence here makes it sound like it's more relevant
The followup line is only really relevant to the bases part of the process.